### PR TITLE
Add an FAQ about asking questions effectively

### PR DESCRIPTION
**Pull request by MCJack123**
This PR adds an FAQ that describes how to effectively write a question that can be answered without having to ask a bunch of follow-up questions. It follows the general structure of well-formed bug reports (specifically inspired by Apple bug reports), and this is how I try to structure questions when I have them. This will help new users be able to explain themselves fully beforehand so they don't have to get frustrated after answering ten different questions about what they want to achieve.

### CODE REVIEW

**Comment by MCJack123** [comment]
I've (finally) chopped it down under 2000 characters, so it should be good to go now.

File: faqs/asking_questions.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: How to Ask a Question
+search: ask question
+---
+Knowing how to effectively ask a question is essential in getting help from others. If you ask a vague question without many details, others have to slowly pry out what exactly you need. By stating everything up front, it's much easier to get answers because those who want to help don't have to ask a bunch of questions just to know what you want.
+
+**1. Give a summary of your problem**  
+State briefly what the issue you're running into is. Explain your goal and what problem is occurring. This should be no more than a few sentences; more detail will be added later.
+
+**2. Show the code or program you're running**  
+This is where you link to or embed the code you are using. If you're writing a program, paste in the code you wrote inside "\`\`\`lua" and "\`\`\`" brackets - this gives it some styling that makes it easier to read. (An example is listed below.) If you're using someone else's program, link the Pastebin or forum URL that you got it from.
```
Comment (SquidDev) [inline faqs/asking_questions.md:11]: > An example is listed below.

I don't think this is still the case is it?

Comment (MCJack123) [reply faqs/asking_questions.md]: Oop, fixed.

**Review by SquidDev — APPROVED** [review]

**Comment by SquidDev** [comment]
Thanks!

**Pull request merged**